upup/pkg/fi/cloudup/awstasks/vpc.go

```diff
@@ -143,11 +144,10 @@ func (_ *VPC) RenderAWS(t *awsup.AWSAPITarget, a, e, changes *VPC) error {
 			if featureflag.VPCSkipEnableDNSSupport.Enabled() {
 				glog.Warningf("VPC did not have EnableDNSSupport=true, but ignoring because of VPCSkipEnableDNSSupport feature-flag")
 			} else {
+				// TODO: We could easily just allow kops to fix this...
```

Comment:
Would be good to automatically rectify this issue but could cause issues if users are not expecting any change to occur to a shared resource. Maybe an auto-fix flag and cover other similar cases if there are any?